synapse/models/inet.py

```diff
@@ -149,20 +164,90 @@ def getDataModel():
             ('inet:netuser',{'ptype':'inet:netuser'},[
                 ('site',{'ptype':'inet:fqdn','ro':1}),
                 ('user',{'ptype':'inet:user','ro':1}),
-                ('name',{'ptype':'str','defval':'??'}),
+
+                ('dob',{'ptype':'time'}),
+
+                #('bio:bt',{'ptype':'wtf','doc':'The netusers self documented blood type'}),
+
+                ('url',{'ptype':'inet:url'}),
+                ('webpage',{'ptype':'inet:url'}),
+                ('avatar',{'ptype':'file:bytes'}),
+
+                ('tagline',{'ptype':'str:txt','doc':'A netuser status/tag line text'}),
+                ('occupation',{'ptype':'str:txt','doc':'A netuser self declared occupation'}),
```

Comment:
Additional common items for consideration as secondary props: birthplace, current location, education / school.